pkg/pool-weighted/contracts/managed/ManagedPoolSwapFees.sol

```diff
@@ -0,0 +1,100 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+import "./ManagedPoolStorageLib.sol";
+
+abstract contract ManagedPoolSwapFees {
```

Comment:
Libraries can have events, so this does not need to be a contract anymore.

Reply:
Unfortunately this results in ManagedPool's ABI not containing these events anymore. We can leave it as, revert, or add dummy events just for the ABI.

Reply:
I want to cry